nixos/modules/services/monitoring/graphite.nix

```diff
@@ -344,49 +340,6 @@ in {
       };
     };
 
-    pager = {
```

Comment:
Do these services no longer exist upstream?

I think you want to add some `mkRemovedOptionModule` statements for these, so that configs fail with an explanation for any current users.

Reply:
Last release for graphite-pager is from 2016 (https://pypi.org/project/graphitepager/), and the upstream github repository does not even exist anymore (https://github.com/seatgeek/graphite-pager).

From this, I think it is safe to assume that this is dead.

I could however have a look at `mkRemovedOptionModule` to properly document it (even if I am pretty sure no one is using it anymore, so it will not break for anyone)

Reply:
done.